xml/core_registers.xml

```diff
@@ -288,6 +288,28 @@
         \end{tabular}
         \end{table}
 
+        Executing the Program Buffer may cause the value of \RcsrDpc to become
+        \unspecified. If that is the case, it must be possible to read/write
+        \RcsrDpc using an abstract command with \FacAccessregisterPostexec not
+        set.  The debugger must attempt to save \RcsrDpc between halting and
+        executing a Program Buffer, and then restore \RcsrDpc before leaving
+        Debug Mode.
+
+        \begin{commentary}
+            Allowing \RcsrDpc to become \unspecified\ upon Program Buffer
+            execution allows for direct implementations that don't have a
+            separate PC register, and do need to use the PC when executing the
+            Program Buffer.
+        \end{commentary}
+
+        If the Access Register abstract command supports reading \RcsrDpc while
+        the hart is running, then the value read should be the address of a
+        recently executed instruction.
+
+        If the Access Register abstract command supports writing \RcsrDpc while
+        the hart is running, then the executing program should jump to the
+        written address shortly after the write occurs.
```

Comment:
The read behavior sounds good but this write behavior seems kind of weird.  Does it seem useful for debuggers to do this or would it be OK to just make writes have UNSPECIFIED behavior?

Reply:
I agree it's weird, but if write is implemented then this is the sensible thing IMO. I wouldn't expect anyone to support writing DPC while running.

Reply:
I haven't thought about abstract reads/writes in a while so I forgot that it's allowable to support reads but not support writes.  This is fine since it has the "if" clause.